config/environment.js

```diff
@@ -67,7 +67,10 @@ module.exports = function(environment) {
 
     // Whether to take the participant to the last page they were on
     // when they exited the study. If false, start from the beginning.
-    continueSession: true
+    continueSession: true,
+
+    // Whether to include the test locale or not
+    excludeTestLocale: environment === 'production',
```

Comment:
Note for followup:

Actually, this is still tied to build settings. "Production" has no relationship to what server we are running on- in point of fact, we should be using `--environment production` to build on staging. This is relevant because sometimes we ask QA to use this translation on staging.

In the future, this should be completely separate from build setting. As discussed a while back, we'd ideally like to move some of these flags into a `.env` file- true environment configuration. Main blocker is that MattC is out of town.